fe/fe-core/src/main/java/com/starrocks/common/Config.java

```diff
@@ -1296,6 +1296,42 @@
     @ConfField
     public static long hive_meta_store_timeout_s = 10L;
 
+    /**
+     * If set to true, StarRocks will automatically synchronize hms metadata to the cache in fe.
+     */
+    @ConfField(mutable = true)
+    public static boolean enable_hms_events_incremental_sync = false;
```

Comment:
Do we really need so many configs?

Reply:
Because we support both manual refresh and automatic refresh, and automatic refresh is likely to have bug during user use. Therefore, adding these configurations is equivalent to providing users some remedial measures. Of course, with the iteration of the feature, for example, some grammars may be provided later to allow users to query the status of such background tasks, and these parameters can be deleted at that time.